server/config/db.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable import/no-mutable-exports */
+import pg from 'pg';
+import dotenv from 'dotenv';
+
+dotenv.config();
```

Comment:
setting up dotenv is done only once. basically in the entry point of your project. then you no longer need to set it up